DPRHTML/dictionary.html

```diff
@@ -12,14 +12,8 @@
         DPR_PAL.closeSideBar();
         $("#mafbc").load("dictionary-results.html");
       });
-      $('#dictinInfo')
-        .each(function() {
-          $(this).popover({
-            trigger: 'click',
-            html : true,
-            content: () => $('#dictinInfo-popover-content').html(),
-          })
-        });
+
+      DPR_PAL.enablePopover('#dictinInfo', 'click');
```

Comment:
Here I also would prefer trigger _hover_ instead of _click_. It's not intuitive to have to click another time on the button to send it away.

Reply:
done